lib/rules/jsx-no-literals.js

```diff
@@ -40,27 +40,40 @@ module.exports = {
         },
         ignoreProps: {
           type: 'boolean'
+        },
+        noAttributeStrings: {
+          type: 'boolean'
         }
       },
+
       additionalProperties: false
     }]
   },
 
   create(context) {
-    const defaults = {noStrings: false, allowedStrings: [], ignoreProps: false};
+    const defaults = {
+      noStrings: false, allowedStrings: [], ignoreProps: false, noAttributeStrings: false
```

Comment:
```suggestion
      noStrings: false,
      allowedStrings: [],
      ignoreProps: false,
      noAttributeStrings: false
```
when the entire object doesn't fit on one line, each property should be on its own line.